CGATPipelines/PipelineGeneset.py

```diff
@@ -523,14 +523,16 @@ def buildCDSFasta(infiles, outfile):
 
     tmpfile = P.getTempFile(".")
 
+    transcript_table = P.toTable(infile_transcripts)
+
     dbhandle = sqlite3.connect(PARAMS["database_name"])
     cc = dbhandle.cursor()
+    statement = '''SELECT DISTINCT protein_id, transcript_id
+                   FROM %(transcript_table)s''' % locals()
+
     tmpfile.write("protein_id\ttranscript_id\n")
     tmpfile.write("\n".join(
-        ["%s\t%s" % x for x in
-         cc.execute(
-             "SELECT DISTINCT protein_id, transcript_id "
-             "FROM transcript_info")]))
```

Comment:
Yes, I am not sure, too. The call signature should be:

def buildCDSFasta(infiles, outfile, transcript_table="transcript_info"):

Best wishes,
Andreas